src/bin/tester.rs

```diff
@@ -136,6 +139,87 @@ fn ambiguity_check(handle: ModuleContentsHandle) -> Result<(), MoreThanOne> {
     Ok(())
 }
 
+#[derive(Debug)]
+struct Counters {
+    total_count: u16,
+    unambiguous_count: u16,
+    ambiguous_count: u16,
+    too_short_count: u16,
+    no_parse_count: u16,
+}
+
+impl Add for Counters {
```

Comment:
Use https://jeltef.github.io/derive_more/derive_more/add.html ?

Reply:
Wow, I didn't know this one! super cool!